src/router_core/modules/test_hooks/core_test_hooks.c

```diff
@@ -585,7 +585,7 @@ static void _do_send(test_client_t *tc)
         assert(rc == 0);
         ++tc->counter;
         --tc->credit;
-        qd_log(LOG_ROUTER_CORE, QD_LOG_DEBUG, "client test message sent id=%" PRIi64 " c=%d", tc->counter - 1,
```

Comment:
One nit: the type for tc->counter is "long", so the proper type specifier would be %ld.

Reply:
I fixed the format specifier of `tc->counter` to `%ld`